paperscraper/get_dumps/utils/chemrxiv/chemrxiv_api.py

```diff
@@ -11,9 +14,44 @@ class ChemrxivAPI:
 
     base = "https://chemrxiv.org/engage/chemrxiv/public-api/v1"
 
-    def __init__(self, page_size: Optional[int] = None):
+    def __init__(self,
+                 begin_date: Optional[str] = None,
+                 end_date: Optional[str] = None,
+                 page_size: Optional[int] = None):
+        """
+        Initialize API class.
+
+        Args:
+            begin_date (Optional[str], optional): begin date expressed as YYYY-MM-DD. 
+                Defaults to None.
+            end_date (Optional[str], optional): end date expressed as YYYY-MM-DD.
+                Defaults to None.
+            page_size (int, optional): The batch size used to fetch the records from chemrxiv.
+        """
 
         self.page_size = page_size or 50
+        
+        # Begin Date and End Date of the search
+        launch_date = launch_dates['chemrxiv']
+        launch_datetime = datetime.fromisoformat(launch_date)
+
+        if begin_date:
+            begin_datetime = datetime.fromisoformat(begin_date)
+            if begin_datetime < launch_datetime:
+                self.begin_date = launch_date
```

Comment:
In this `if` clause, let's make a `logger.warning` to inform the user that the date cant be before the launch date. We should not silently alter the input date as this might confuse the user